middleware/buffer/source/field.cpp

```diff
@@ -170,7 +170,7 @@ namespace casual
                // called on service invocation
                common::buffer::handle::mutate::type adopt( common::buffer::Payload&& payload, common::buffer::handle::type* inbound)
                {
-                  Trace trace{ "buffer::field::Allocator::adopt"};
+                  //Trace trace{ "buffer::field::Allocator::adopt"};
                   auto& buffer = allocator_base::emplace_back( std::move( payload), inbound);
```

Comment:
Should Trace be commented? If this is intentional, should the line be removed instead?

Reply:
I guess your analysis is correct